src/remote.ml

```diff
@@ -1581,20 +1581,20 @@ let halfduplex =
      simultaneously emit data).  If you experience unstabilities with \
      your network link, this may help."
 
-let negociateFlowControlLocal conn () =
+let negotiateFlowControlLocal conn () =
   disableFlowControl conn.outputQueue;
   Lwt.return false
 
-let negociateFlowControlRemote =
-  registerServerCmd "negociateFlowControl" Umarshal.unit Umarshal.bool negociateFlowControlLocal
+let negotiateFlowControlRemote =
+  registerServerCmd "negotiateFlowControl" Umarshal.unit Umarshal.bool negotiateFlowControlLocal
```

Comment:
This one is a no-go. The string argument is part of the RPC API and must not be changed.